_includes/testimonials.html

```diff
@@ -4,12 +4,20 @@ <h2 class="section-heading text-center">Testimonials</h2>
         <h3 class="section-subheading text-muted text-center">For confidentiality, some details are omitted</h3>
     </div>
 
+    <div id="testimonials-carousel" class="carousel slide" data-ride="carousel">
```

Comment:
with slick you can just create the buttons outside of the carosel and pass the element to the slick call:

https://stackoverflow.com/questions/29043562/add-custom-buttons-on-slick-carousel

Reply:
maybe we can redelete the fonts in font if we use slick.

Reply:
I had a conflict, since we worked on this file at the same time. Did I delete something necessary?